plugins/importexport/native/filter/NativeXmlPublicationFormatFilter.inc.php

```diff
@@ -88,6 +89,26 @@ function handleElement($node) {
 	}
 
 	/**
+	 * Process the self_file_ref node found inside the publication_format node.
+	 * @param $node DOMElement
+	 * @param $deployment Onix30ExportDeployment
+	 * @param $representation PublicationFormat
+	 */
+	function _processFileRef($node, $deployment, &$representation) {
```

Comment:
FYI, we don't need to use references on objects anymore; since PHP5, object variables are references.